test/fgpu/test_engine.py

```diff
@@ -956,6 +956,40 @@ async def test_dig_clip_cnt_sensors(
         assert [r.value for r in sensor_update_dict[sensors[1].name]] == [0, 0] + [10000] * 7
         assert [r.timestamp for r in sensor_update_dict[sensors[1].name]] == expected_timestamps
 
+    # It's easier to use a constant voltage. Also need to check the case were
+    # the input power is zero.
+    @pytest.mark.parametrize("inout", [(0, np.finfo(np.float64).min), (64, -15.034518566376697)])
```

Comment:
It might be worth using something like `pytest.approx(-15.0345186)` so that a change in the implementation that causes slightly different rounding doesn't break the test. You'd just need to check that the test ID (in `pytest -v`) still looks sane.

By the way, how is that magic value derived? Did you calculate it (in which case it might make sense to put the calculation in the code to make it self-documenting) or just observe what the value happened to be?

Reply:
I started off trying to calculate it, but the fact that it was Friday afternoon got to me and I decided just to run through and see what value came out. Then I made note of it.

The parameterisation looks like this:
```
test/fgpu/test_engine.py::TestEngine::test_dig_rms_dbfs_sensors[wideband-NVIDIA GeForce RTX 3080 Ti (CUDA)-0--1.7976931348623157e+308] PASSED            [ 25%]
test/fgpu/test_engine.py::TestEngine::test_dig_rms_dbfs_sensors[wideband-NVIDIA GeForce RTX 3080 Ti (CUDA)-64-output_power_dbfs1] PASSED                 [ 50%]
test/fgpu/test_engine.py::TestEngine::test_dig_rms_dbfs_sensors[narrowband-NVIDIA GeForce RTX 3080 Ti (CUDA)-0--1.7976931348623157e+308] PASSED          [ 75%]
test/fgpu/test_engine.py::TestEngine::test_dig_rms_dbfs_sensors[narrowband-NVIDIA GeForce RTX 3080 Ti (CUDA)-64-output_power_dbfs1] PASSED               [100%]
```
which seems OK to me.